js/ui/iconGrid.js

```diff
@@ -68,9 +69,24 @@ const BaseIcon = new Lang.Class({
         this.iconSize = ICON_SIZE;
         this._iconBin = new St.Bin({ x_align: St.Align.MIDDLE,
                                      y_align: St.Align.MIDDLE });
+        this._iconBin.add_style_class_name('icon-button');
```

Comment:
Note that in eos-shell this is actually `iconButton` and is an `St.Button`. I'm not sure why that was, so I've left it as `St.Bin` for now.